wbooks/src/app/screens/BookDetail/components/CommentTable/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import {  
+  Text,
+  View,
+  FlatList,
+  TouchableOpacity
+} from 'react-native';
+
+import CommentItem from './component/CommentItem';
+import styles from './styles';
+import COMMENTS from '../../comments';
+
+class CommentTable extends Component {
+
+  renderItem = ({ item }) => (
+    <TouchableOpacity 
+      onPress={() => {}}
+    >
```

Comment:
i think we won't need the comment to be touchable

Reply:
done